### PR TITLE
implemented CRUD operations for the customer class

### PR DESCRIPTION
**Pull request by tshet-eg**

### CODE REVIEW

File: InvoiceManagementSystem/Repositories/Customer.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                if (CustomerID == customerEntry.CustomerID)
+                {
+                    isPresent = true;
+                    break;
+                }
+            }
+            return isPresent;
+        }
+
+        public Models.Customer GetCustomer(string CustomerID)
```
Comment (sheetal2605) [inline InvoiceManagementSystem/Repositories/Customer.cs]: DisplayCustomerDetails() and GetCustomer() functions has the same implementation.

File: InvoiceManagementSystem/Repositories/Customer.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            {
+                if (CustomerID == customerEntry.CustomerID)
+                {
+                    customer = customerEntry;
+
+                }
+            }
+            return customer;
+        }
+        public bool CheckCustomer(string CustomerID)
```
Comment (sheetal2605) [inline InvoiceManagementSystem/Repositories/Customer.cs]: Add CheckCustomer() to validations folder

**Review by sheetal2605 — CHANGES_REQUESTED** [review]

File: InvoiceManagementSystem/Repositories/Customer.cs (hunk 1 above)
Comment (tshet-eg) [reply InvoiceManagementSystem/Repositories/Customer.cs]: will look

**Review by sheetal2605 — APPROVED** [review]
good to go

**Pull request merged**